Bottom: fc91142e55ce36cec0635eed5fd64fcab7511f7d
Top:    eafab15008dd6b6c5bbabe139fea56903c38e89f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 15:56:01 +0000

Mark segments as not round before falling back to line-line intersection

This avoids assert failures with debugging enabeled


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 401abc9..adb33e4 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1325,6 +1325,8 @@ seg_in_seg_arc_arc (struct info *i, struct seg *s1, struct seg *s2)
       s1->v->radius == s2->v->radius)
     {
       printf ("Cop-out for co-circular arcs for now - they will be better handled by their line approximations. <<<<<<<<<<\n");
+      s1->v->is_round = false;
+      s2->v->is_round = false;
       return seg_in_seg_line_line (i, s1, s2);
     }
 #endif
